declarative-examples/booleanParamBuildStep.groovy

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@rsandell - right. Okay, then we need to audit this whole repo an apply licenses to all the files. 😄